app/api/v2/__init__.py

```diff
@@ -0,0 +1,29 @@
+from flask import Blueprint
+from flask_restful import Api
+
+# Defines the blueprint application
+# Local imports
+# from app.api.v1.views.products import Product, Products
+# from app.api.v1.views.sales import Sale, Sales
+from app.api.v2.views.users import (Signup, Login)
```

Comment:
Get rid of these unnecessary folders like api and v1, move what is in v1 to app folder and get tests folder outside the app folder

Reply:
Hello Boniface,
I have taken note of this feedback.
I would request to hold this for the last. Doing so will result in breaking the codes and throw me into immediate chaos.
Is there an alternative to this feedback you have made?

Regards
Steven Ennis.